rule-audit-client/src/constants.ts

```diff
@@ -0,0 +1,5 @@
+const urls = {
+    capiQueryUrl: 'capi'
```

Comment:
How does `'capi'` get used? Should it be a URL?

Reply:
This would be better named as `capiQueryPath` – it's the path to query the capi controller, mirrored in `conf/routes` in the main project.

It'd be nice to derive both the paths and the types from the play project, wouldn't it? 😁